include/dd/ComplexValue.hpp

```diff
@@ -56,18 +57,20 @@ namespace dd {
             os.write(reinterpret_cast<const char*>(&i), sizeof(decltype(i)));
         }
 
-        void from_string(const std::string& real_str, std::string imag_str) {
-            fp real = real_str.empty() ? 0. : std::stod(real_str);
+        void fromString(const std::string& realStr, std::string imagStr) {
+            fp real = realStr.empty() ? 0. : std::stod(realStr);
 
-            imag_str.erase(remove(imag_str.begin(), imag_str.end(), ' '), imag_str.end());
-            imag_str.erase(remove(imag_str.begin(), imag_str.end(), 'i'), imag_str.end());
-            if (imag_str == "+" || imag_str == "-") imag_str = imag_str + "1";
-            fp imag = imag_str.empty() ? 0. : std::stod(imag_str);
+            imagStr.erase(remove(imagStr.begin(), imagStr.end(), ' '), imagStr.end());
+            imagStr.erase(remove(imagStr.begin(), imagStr.end(), 'i'), imagStr.end());
+            if (imagStr == "+" || imagStr == "-") {
+                imagStr = imagStr + "1";
+            }
+            fp imag = imagStr.empty() ? 0. : std::stod(imagStr);
             r       = {real};
             i       = {imag};
         }
 
-        static auto getLowestFraction(const double x, const std::uint64_t maxDenominator = std::uint64_t(1) << 10, const fp tol = dd::ComplexTable<>::tolerance()) {
+        static auto getLowestFraction(const double x, const std::uint64_t maxDenominator = static_cast<std::uint64_t>(1) << 10, const fp tol = dd::ComplexTable<>::tolerance()) {
```

Comment:
Just a nitpick. This might be cleaner as
```suggestion
        static auto getLowestFraction(const double x, const std::uint64_t maxDenominator = 1U << 10, const fp tol = dd::ComplexTable<>::tolerance()) {
```
Or does this raise an implicit conversation warning again?